pkg/csi/service/common/vsphereutil.go

```diff
@@ -645,3 +646,40 @@ func isExpansionRequired(ctx context.Context, volumeID string, requestedSize int
 
 	return currentSize < requestedSize, nil
 }
+
+// CreateSnapshotUtil is the helper function to create CNS snapshot for given volumeId
+// the desc parameter denotes the snapshot description required by CNS CreateSnapshot API.
```

Comment:
Is the description field required by CNS CreateSnapshot API or is it optional?

Reply:
It is required but without any validation in its format.